File: Excercices 1-50/C++/1_Multiples_of_3_or_5.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

/*
EXCERCISE 1: Multiples of 3 and 5
If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3,5,6 and 9. 
The sum of these multiples is 23.
Find the sum of all the multiples of 3 or 5 below 1000. */

int main(){

    int sum = 0;

    for (int i = 0; i < 1000; i++){
        if (i % 3 == 0 || i % 5 == 0){
            sum += i;
        }
    }
    printf("%d\n", sum);//ANS:233168

}

File: Excercices 1-50/Python/1.Multiples of 3 or 5.py
"""
If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3,5,6 and 9. The sum of these multiples is 23.
Find the sum of all the multiples of 3 or 5 below 1000.
"""

def main():

    list_of_multiples = []

    for i in range(1, 1000):

        if i % 3 == 0 or i % 5 == 0:

            list_of_multiples.append(i)

    print(sum(list_of_multiples)) #ANS:233168
       
if __name__ == "__main__":
    main()